### PR TITLE
Fix a typo and remove redundancies

### PR DESCRIPTION
**Pull request by atymchuk**

### CODE REVIEW

File: .github/workflows/pipeline.yaml [hunk 1]
```diff
@@ -1,15 +1,12 @@
 name: Build and Deploy
 on:
-  pull_request:
```
Comment (atymchuk) [inline .github/workflows/pipeline.yaml:3]: This was was triggering the build twice, which is redundant and gives no benefit.

File: .github/workflows/pipeline.yaml [hunk 2]
```diff
@@ -1,15 +1,12 @@
...
-    branches-ignore:
-      - gh-pages
   push:
     branches-ignore:
       - gh-pages
 jobs:
   build:
     name: Build
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```
Comment (atymchuk) [inline .github/workflows/pipeline.yaml:9]: Latest is somewhat equivalent to auto maintenance )

**Comment by atymchuk** [comment]
It also addresses the issue #197

**Review by dbazhenov — APPROVED** [review]
Great

**Review by LenzGr — APPROVED** [review]
LGTM, thank you!

**Pull request merged**